pkg/controller/githubsource/reconcile.go

```diff
@@ -95,9 +95,12 @@ func (r *reconciler) reconcile(ctx context.Context, source *sourcesv1alpha1.GitH
 	uri, err := sinks.GetSinkURI(ctx, r.client, source.Spec.Sink, source.Namespace)
 	if err != nil {
```

Comment:
Should we check the error for NotFound and not bail on any error?

Reply:
`GetSinkURI` also returns an error when sink ref is nil which can be ignored in this deleted case. Although I doubt this scenario could ever happen when the finalizer is set (since we would have previously validated sink ref existed while adding the finalizer)